### PR TITLE
Expand CI

### PR DESCRIPTION
**Pull request by thomwiggers**
Test newer Django versions

### CODE REVIEW

**Comment by thomwiggers** [comment]
De pull-request die je wist dat zou komen (~hacktoberfest)

File: testprojects/django111/testproject/settings.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+"""
+Django settings for testproject project.
+
+Generated by 'django-admin startproject' using Django 1.10.
```
Comment (joostrijneveld) [inline testprojects/django111/testproject/settings.py:4]: Are you sure this is a Django 1.11 project?

File: testprojects/django20/testproject/settings.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+"""
+Django settings for testproject project.
+
+Generated by 'django-admin startproject' using Django 1.10.
```
Comment (joostrijneveld) [inline testprojects/django20/testproject/settings.py:4]: Same thing as above; is this actually 2.0?

**Comment by joostrijneveld** [comment]
Could you also update the `README.md` accordingly?

**Comment by thomwiggers** [comment]
Done.

**Review by joostrijneveld — CHANGES_REQUESTED** [review]

File: testprojects/django111/testproject/settings.py (hunk 1 above)
Comment (thomwiggers) [reply testprojects/django111/testproject/settings.py:4]: Well, no, it's been copied from 1.10. I don't think that much has changed, actually.

File: testprojects/django20/testproject/settings.py (hunk 2 above)
Comment (thomwiggers) [reply testprojects/django20/testproject/settings.py:4]: same.

**Comment by thomwiggers** [comment]
Not sure why Github isn't updating, but I've updated the testprojects to be generated by the respective Django versions.

**Review by joostrijneveld — APPROVED** [review]

**Comment by joostrijneveld** [comment]
That's weird. When I look at the contents of the branch, your edit does show up. Probably [some cache](https://xkcd.com/908/) somewhere?

**Comment by thomwiggers** [comment]
They didn't make it into the actual merge afaict though. PR #11 has a diff!

**Pull request merged**